packages/react-sdk/docusaurus/docs/React/01-basics/07-ui-customization-options.mdx

```diff
@@ -17,19 +17,19 @@ Here is a list of things still to address on this page:
 There are different ways to use the React Video SDK to build out UI. It comes with powerful built-in video components ready for use.
 In addition, it allows for styling and custom theming options with custom CSS.
 
-For custom use cases it is also possible to only use the low-level client to provide completely custom UI to fully tailor the SDK towards your needs.
+For custom use cases it is also possible to create your own components to fully tailor the SDK towards your needs. Please note that we suggest not to override a few, core components as those contain complex, low-level logic related to performance, the list of these components can be found in the [UI Components overview](../03-ui/01-overview.mdx).
 
 This page guides you through all of them with increasing levels of customization.
```

Comment:
What the adverb "them" refers to?

Reply:
hm, I can't seem to find "them" in the text above, if you mean the "the list of these" that refers to the core components referenced in the previous part of the sentence, it seemed to clear to me when I was writing it

Reply:
This page guides you through all of **them** with increasing levels of customization.

Reply:
Ah, I looked only to my changes, that line wasn't written by me, but from the context them refers to the different customization levels.

Reply:
With more info in the paragraph before we should maybe change this for more clarity:

```suggestion
This page guides you through all the options you have with increasing levels of customization.
```